cpu/kinetis/vectors.c

```diff
@@ -187,31 +187,70 @@ ISR_VECTOR(1) const isr_t vector_cpu[CPU_IRQ_NUMOF] = {
     [I2S0_Rx_IRQn    ] = isr_i2s0_rx,         /* I2S0 receive interrupt */
 #endif
 #ifdef UART0
-#ifdef UART_RPL_RPL_MASK
+#  ifdef KINETIS_SINGLE_UART_IRQ
```

Comment:
we might need to discuss and clarify (in coding conventions) how to indent pre-processor `#if`s

Reply:
What would be your preferred indentation. Most of the code doesn't indent nested `#if`s, but sometimes it gets too hard to read.